playbooks/roce/vf-activate.yaml

```diff
@@ -0,0 +1,18 @@
+---
+# playbook for VF activation after installing  Mellanox OFED
+#
+# Author: Vitaliy Razinkov <vitaliyra@mellanox.com>
+#
+
+- name: "Check activation"
```

Comment:
My test deployment initially failed because there was no /etc/rc.local script present on my nodes.  I had to create an empty /etc/rc.local script on each node, make it executable, and then restart the rc.local service in order to get this playbook to complete succesfully.  I would suggest adding a step that checks for the existence of /etc/rc.local and then creates it if one doesn't exist.

Reply:
I add file checking.

Reply:
Thanks Michael for your suggestion. It's my first experience with Ansible. I will be glad for any comments.